views/political_information.py

```diff
@@ -1,10 +1,43 @@
 from rest_framework import generics, permissions
 from rest_framework.exceptions import NotFound
-
+from rest_framework.response import Response
+from rest_framework import status
+from django_countries import countries
 from omniport.utils.switcher import load_serializer
+from shell.constants import reservation_categories
+from django.http import HttpResponseBadRequest
+import json
+
+
+def to_representation(choices):
```

Comment:
There is a function called `self.get_<name of the choice field>_display` auto created by Django which gets the display name of the choice. `self` here points to an instance of the model where this reservation category is a field.